signalfx-org-metrics/README.md.jinja

```diff
@@ -76,13 +76,12 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
    accepting data points for existing containers. To monitor your usage against the limit, use the metric
    `sf.org.numResourcesMonitored` and filter for the dimension `resourceId:containers`.
 
-* `sf.org.limit.customMetricTimeSeries` (gauge): Maximum number of active custom MTS, within a moving window of the
-   previous 60 minutes, that you're allowed to have in your organization. If you exceed this limit,
-   Infrastructure Monitoring stops accepting data points for new custom MTS, but it continues to accept
-   data points for custom MTS that already existed. To see the number of MTS you're allowed to have, use the metric
-   `sf.org.numCustomMetrics`. To learn more about custom MTS, see the section
-   [About custom, bundled, and high-resolution metrics](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)
-   in the user documentation.
+* `sf.org.limit.customMetricTimeSeries` (gauge): Maximum number of active custom MTS,
+   within a moving window of the previous 60 minutes, that can send data to your organization.
+   If you exceed this limit, Infrastructure Monitoring drops data points for the custom MTS
+   that exceeded the limit, but it continues to accept data points for custom MTS that already existed.
+   you’ve defined, use the metric `sf.org.numCustomMetrics`. To learn more about custom MTS, see the
```

Comment:
The phrase "you’ve defined, use the metric `sf.org.numCustomMetrics`." is not a sentence.